.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -9,6 +9,7 @@ assignees: ''
 
 # Prepare for the Release
 
+- [ ] Make sure there has been [at least one full sync test](https://github.com/ZcashFoundation/zebra/actions/workflows/continous-integration-docker.yml?query=event%3Aschedule) since the last state change, or start a manual full sync.
```

Comment:
```suggestion
- [ ] Make sure there has been [at least one successful full sync test](https://github.com/ZcashFoundation/zebra/actions/workflows/continous-integration-docker.yml?query=event%3Aschedule) since the last state change, or start a manual full sync.
```

If you apply the previous one you might want to do this one as well.